Gamma_Analysis.py

```diff
@@ -234,7 +240,12 @@ def make_table(isotope_list, sample_info, sample_names, dates):
             mass[j] = float(mass[j])
         mass[j] = 1000/mass[j]
     for i in range(len(sample_names)):
-        data[sample_names[i]] = np.array(sample_info[i]) * mass[i]
+        information = []
+        for k in range(len(sample_info[i])):
```

Comment:
```
for inf in sample_info[i]:
    value = np.array(inf * mass[i]).round(decimals=2)
    information.append(value)
```
or even
```
values = np.array(sample_info[i]) * mass[i]
data[sample_names[i]] = values.round(decimals=2)
```
except that, as discussed, instead of just rounding to 2 decimals every time, this should depend on the uncertainty. But I give you these examples to illustrate simpler, more pythonic ways you could do this.